test/browser-only/__snapshots__/constructLayoutEngine-browser.test.js.snap

```diff
@@ -392,17 +392,27 @@ exports[`constructLayoutEngine browser error handling works with error handlers
 <body>
   <div
     id="single-spa-application:header"
-  />
+  >
+    <div>
+      Oops! An error occurred!
+    </div>
+  </div>
   <div
     class="main-content"
   >
     <div
       id="single-spa-application:app1"
     >
-      <div>
-        Oops! An error occurred!
-      </div>
+      error parcel mounted
```

Comment:
This changed because I changed the implementation of the error UI for this application